frontend/src/flashcard.tsx

```diff
@@ -63,31 +128,27 @@ const Flashcard: React.FC<FlashcardProps> = ({
         // Front side
         <Stack align="center" gap="md">
           <Group>
-            <Button
-              variant="transparent"
-              size="xs"
-              c={isDark ? theme.colors.gray[8] : theme.colors.gray[5]}
-              onClick={onShuffle}
-            >
-              <IconArrowsShuffle />
-            </Button>
-            <Text
+            {!smallScreen && shuffleButton}
+            <TiledText
               size="xl"
```

Comment:
Would be interested in having the text size here exceed the `xl` option, not sure how frowned upon it is to just hard-code a larger size